src/site/paragraphs/react_widget.drupal.liquid

```diff
@@ -40,7 +40,11 @@
       {% if entityUrl.path contains "/eligibility" %}data-widget-show-learn-more{% endif %}
       data-widget-timeout="{{entity.fieldTimeout}}">
       <div class="loading-indicator-container">
-        <div class="loading-indicator" role="progressbar" aria-valuetext="{{ entity.fieldLoadingMessage }}"></div>
+        <div
+          aria-label="{% if entity.fieldLoadingMessage %}{{ entity.fieldLoadingMessage }}{% else %}loading indicator{% endif %}"
```

Comment:
I'd like to match the React component for default language if no `entity.fieldLoading Message`. Can we change "loading indicator" string to "Loading..." ?

https://design.va.gov/storybook/?path=/docs/components-loadingindicator--default